src/components/existing-request/existing-request-display.vue

```diff
@@ -196,10 +196,15 @@
             :emailAddress="nr && nr.applicants && nr.applicants.emailAddress"
             :showIncorporateNowButton="showIncorporateButton"
             :showRegisterButton="showRegisterButton"
+            :showAlterNowButton="showAlterNowButton"
+            :isAllowAlterOnline="isAllowAlterOnline"
+            :isOpenExternal="isOpenExternal"
             :showGoToSocietiesButton="showGoToSocietiesButton"
```

Comment:
This is now getting messy (too many props that mean almost the same thing). Could their logic be moved into NrApprovedGrayBox instead (ie, make the subcomponent smarter so we don't have to pass in so many props)?

Reply:
This will be handled in bcgov/entity#17608.